sherpa-onnx/csrc/provider-config.h

```diff
@@ -67,17 +67,17 @@ struct TensorrtConfig {
 };
 
 struct ProviderConfig {
+  TensorrtConfig trt_config;
+  CudaConfig cuda_config;
+  std::string provider = "cpu";
   uint32_t device = 0;
```

Comment:
By the way, is there a reason to use `uint32_t` instead of `int32_t`?

Reply:
Nothing special, just went with it,
want to keep it consistent with repo, as int32_t?
did think about compatibility with other devices.

Reply:
If there is no special reason, please using `int32_t`.

unsigned integers is error-prone.